keyboards/ztboards/noon/info.json

```diff
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "ZTBoards Noon", 
+    "url": "ztboards.myshopify.com", 
+    "maintainer": "phlopsuh", 
+    "width": 16, 
+    "height": 5, 
+    "layouts": {
+        "LAYOUT": {
```

Comment:
```suggestion
        "LAYOUT_all": {
```

Also, there are 65 keys noted in this file, but the keymaps and `noon.h` all give 67, though that seems to be inconsistent with the info given on the geekhack thread in the readme.

Reply:
I think it may have been due to the fact that there's a switch spot on my PCB, but our keyboard that the PCB goes into has a blocker in that spot. So I just went ahead and deleted that in the files. I think the numbers should line up now? 

I have split BS support so that could explain the 1 key difference I think.

Reply:
Does the PCB support three 1.25u keys at the bottom left corner? I'm getting the following when I load your default keymap into your `info.json` file in a dev environment:

![image](https://user-images.githubusercontent.com/18669334/89574750-7f34c680-d7e1-11ea-88c3-60a4cf21fa48.png)

Reply:
Ah let me fix that as well. So the PCB supports 3 1.25u keys, but our case won't support it at all.